skbio/stats/ordination/tests/test_ordination_results.py

```diff
@@ -75,6 +75,47 @@ def test_str(self):
                                     pd.Series(np.array([4.2])), samples_df))
         self.assertEqual(obs.split('\n'), exp.split('\n'))
 
+    def test_rename(self):
```

Comment:
Could `test_rename_by_function` be added to assert behavior when providing a function?

Reply:
Done!